src/main/java/nl/tudelft/ewi/devhub/server/database/entities/issues/AbstractIssue.java

```diff
@@ -71,6 +77,18 @@
 	@Temporal(TemporalType.TIMESTAMP)
 	private Date closed;
 
+	// Left away @NotEmpty for now, since existing tests should be updated for it to work
```

Comment:
Can we bring this annotation back? As there is currently no other mechanism to prevent empty title names, which I think is relatively unpreferable.

The current naming schema for pull requests is `Pull Request #${pullRequest.issueId}: ${pullRequest.branchName}` (and hardcoded in the Freemarker templates). Perhaps it is an idea to persist this automatically generated name in the `PullRequestsBackend` and also use the `${pullRequest.title}` field from Freemarker from now on. (Until we also allow the pull request title to be set manually).